### PR TITLE
{devel}[GCCcore/11.2.0] CVS v1.11.23

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/0cd0bae4c3e6c1a1a5d11ca16efb26d2 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/110b500d297ce483ca437a143d880134 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/40a3da35301a8b28792437c4262fd5db for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a437c40a74e04306c017f45767f7603d for a full test report.

File: easybuild/easyconfigs/c/CVS/CVS-1.11.23-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+version = '1.11.23'
+
+homepage = 'https://savannah.nongnu.org/projects/cvs'
+description = """CVS is a version control system, an important component of
+Source Configuration Management (SCM).
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [' http://ftp.gnu.org/non-gnu/cvs/source/stable/%(version)s/']
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CVS/CVS-1.11.23-GCCcore-11.2.0.eb:18]: @SebastianAchilles is there an `https` version of this?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CVS/CVS-1.11.23-GCCcore-11.2.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/c/CVS/CVS-1.11.23-GCCcore-11.2.0.eb]: Yes, `https` works! Changed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14283/commits/134b3a7fc21e3592177ed79154a15d030e4c3e62 Thanks for spotting this 👍

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download"

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/e2400a908e57a3720945ce87bc9f8b0f for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**